main/java/sorts/BubbleSort.java

```diff
@@ -0,0 +1,94 @@
+package sorts;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class BubbleSort implements Sort {
```

Comment:
Interface names are adjectives/nouns [See common Java interface names, Cloneable, Iterable, Comparable]
This would suit more to Sortable